.clang-format

```diff
@@ -0,0 +1,168 @@
+---
+Language:        Cpp
+# BasedOnStyle:  Google
```

Comment:
Why?

Reply:
Haico picked this in his formatting folder in roboteam_suite so I thought let's stick to it. If you prefer a different style, let me know 😇

Reply:
It's commented out?

Reply:
Yes, since the clang-tidy command generates this file based on the Google style, so you can still change/see the different settings, instead of looking online. I just did `clang-tidy style=google`